### PR TITLE
Lock fix for Rhino 8

### PR DESCRIPTION
**Pull request by adamhathcock**
Rhino 8 API is only build for net48?

https://www.nuget.org/packages/RhinoCommon/8.9.24194.18121#supportedframeworks-body-tab

### CODE REVIEW

File: Connectors/Rhino/Speckle.Connectors.Rhino7/Speckle.Connectors.Rhino7.csproj [hunk 1]
```diff
@@ -11,7 +11,6 @@
   <ItemGroup>
     <PackageReference Include="RhinoCommon" IncludeAssets="compile;build" VersionOverride="7.13.21348.13001" />
     <PackageReference Include="RhinoWindows" IncludeAssets="compile;build" VersionOverride="7.13.21348.13001" />
-    <PackageReference Include="System.Resources.Extensions" />
```
Comment (oguzhankoral) [inline Connectors/Rhino/Speckle.Connectors.Rhino7/Speckle.Connectors.Rhino7.csproj:14]: I guess you are right, previously we were using some `.resx`. Just checkin that this library were there bc of this reason?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -24,7 +24,7 @@ jobs:
           key: ${{ runner.os }}-nuget-${{ hashFiles('**/packages.lock.json') }}
 
       - name: ⚒️ Run build
-        run: ./build.sh test-only
```
[the remark this answers is not in the document]
Comment (adamhathcock) [reply .github/workflows/ci.yml:27]: needed to fix build

File: Connectors/Rhino/Speckle.Connectors.Rhino7/Speckle.Connectors.Rhino7.csproj (hunk 1 above)
Comment (adamhathcock) [reply Connectors/Rhino/Speckle.Connectors.Rhino7/Speckle.Connectors.Rhino7.csproj:11]: I don't think we need this at all....there are no resx files.

Comment (oguzhankoral) [reply Connectors/Rhino/Speckle.Connectors.Rhino7/Speckle.Connectors.Rhino7.csproj:11]: nice

**Review by oguzhankoral — APPROVED** [review]

**Review by JR-Morgan — APPROVED** [review]

**Pull request merged**